src/earn/poolInfoScreen/Cards.tsx

```diff
@@ -304,23 +311,29 @@ export function AgeCard({
   return (
     <View style={styles.card} testID="AgeCard">
       <View style={styles.cardLineContainer}>
-        <View style={styles.cardLineLabel}>
+        <View style={styles.flexLabel}>
           <LabelWithInfo
             onPress={onInfoIconPress}
             label={t('earnFlow.poolInfoScreen.ageOfPool')}
-            labelStyle={styles.cardTitleText}
+            labelStyle={styles.lineLabel}
             testID="AgeInfoIcon"
+            numberOfLines={undefined}
```

Comment:
This is the default, right?

Reply:
Removed in fd54594b95927ee10f65a25eef60b1b92c9ced33!